cpu/cc2538/radio/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_CC2538_RF
+    bool "CC2538 IEEE 802.15.4 radio" if !MODULE_NETDEV_DEFAULT
+    depends on TEST_KCONFIG
+    depends on CPU_FAM_CC2538
+    default MODULE_NETDEV_DEFAULT
+    select MODULE_NETDEV_IEEE802154_SUBMAC if MODULE_NETDEV && !MODULE_CC2538_RF_NETDEV_LEGACY
```

Comment:
I think this was removed not long ago.
@jia200x would you mind taking a look at this and the netdev + submac part?